app/actualize_script.php

```diff
@@ -43,44 +54,35 @@
 foreach ($users as $user) {
 	if (($user !== $system_conf->default_user) &&
 			(FreshRSS_UserDAO::mtime($user) < $min_last_activity)) {
-		Minz_Log::notice('FreshRSS skip inactive user ' . $user, ADMIN_LOG);
-		if (defined('STDOUT')) {
-			fwrite(STDOUT, 'FreshRSS skip inactive user ' . $user . "\n");	//Unbuffered
-		}
+		notice('FreshRSS skip inactive user ' . $user);
 		continue;
 	}
-	Minz_Log::notice('FreshRSS actualize ' . $user, ADMIN_LOG);
-	if (defined('STDOUT')) {
-		fwrite(STDOUT, 'Actualize ' . $user . "...\n");	//Unbuffered
-	}
-	echo $user, ' ';	//Buffered
-
 
 	Minz_Session::_param('currentUser', $user);
 	new Minz_ModelPdo($user);	//TODO: FIXME: Quick-fix while waiting for a better FreshRSS() constructor/init
 	FreshRSS_Auth::giveAccess();
 	$app->init();
+	notice('FreshRSS actualize ' . $user . '...');
```

Comment:
This line is moved further down, after user initialisation, otherwise the log line was prefixed with the wrong username (from the previous loop)